setup/install/providers/kubernetes-v2/gke/index.md

```diff
@@ -0,0 +1,61 @@
+---
+Layout: single
+title:  "Google Kubernetes Engine (GKE) Setup"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+This page describes how to set up a Kubernetes cluster on
+[GKE](https://cloud.google.com/kubernetes-engine/) that Spinnaker can
+manage and deploy workloads to. The process is very simple, but some
+considerations need to be made when configuring how Spinnaker will authenticate
+against your cluster.
+
+> Note: To manage and create clusters in a given project, you will need the
+> `roles.container/admin` role as described
+> [here](https://cloud.google.com/kubernetes-engine/docs/how-to/iam#predefined).
+
+# Create a cluster
+
+You can create a Kubernetes cluster on GKE using either
```

Comment:
If you don't already have a cluster for this purpose, you can create one on GKE using either...